src/scmrepo/git/lfs/client.py

```diff
@@ -64,11 +66,12 @@ async def get_client(**kwargs):
                     sock_connect=self._REQUEST_TIMEOUT,
                     sock_read=self._REQUEST_TIMEOUT,
                 ),
-                retry_options=ExponentialRetry(
+                retry_options=_ExponentialRetry(
                     attempts=self._SESSION_RETRIES,
                     factor=self._SESSION_BACKOFF_FACTOR,
                     max_timeout=self._REQUEST_TIMEOUT,
                     exceptions={aiohttp.ClientError},
+                    statuses={HTTPStatus.TOO_MANY_REQUESTS},
```

Comment:
what was the default for this? was it retrying on all codes?

Reply:
The default value is `{}`, it was retrying only on responses with 5xx status codes.

* [`statuses` defaults to `None`](https://github.com/inyutin/aiohttp_retry/blob/ba2169891f5b32a5c59e48ca185dd8e68e44ded7/aiohttp_retry/retry_options.py#L45) and [corresponds to an empty set](https://github.com/inyutin/aiohttp_retry/blob/ba2169891f5b32a5c59e48ca185dd8e68e44ded7/aiohttp_retry/retry_options.py#L22-L23).
* [Requests are retried when the response status code is a member of the `statuses` set](https://github.com/inyutin/aiohttp_retry/blob/ba2169891f5b32a5c59e48ca185dd8e68e44ded7/aiohttp_retry/client.py#L90-L91).
* [Requests with 5xx response status codes are retried when `retry_all_server_errors` is `True`](https://github.com/inyutin/aiohttp_retry/blob/ba2169891f5b32a5c59e48ca185dd8e68e44ded7/aiohttp_retry/client.py#L87-L88), which is the [default value](https://github.com/inyutin/aiohttp_retry/blob/ba2169891f5b32a5c59e48ca185dd8e68e44ded7/aiohttp_retry/retry_options.py#L47).

Reply:
do, just to double check - it will keep retrying on 5xx the same way as we before, we only add new codes to retry on, right?

this kind of things would be nice to have tests for (mocks, not integration)

Reply:
Yes, that's correct.